### PR TITLE
Add hand drawn tile

### PR DESCRIPTION
**Pull request by Ploypiti**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

### Changes done in this Pull Request

- http://rawgit.com/ploypiti/labyrinth/master/index.html

I don't even know if I'm too late for GCI :/

### CODE REVIEW

**Comment by abishekvashok** [comment]
Can you check Travis for lint errors?

**Comment by Ploypiti** [comment]
@abishekvashok No clue.

**Comment by abishekvashok** [comment]
Wait

**Comment by abishekvashok** [comment]
@Ploypiti at vermaTation.js:
vermaTation.js: line 19, col 88, Missing semicolon.

**Comment by niccokunzmann** [comment]
This is also in the master branch. We can ask @Ploypiti to fix it but it is nothing newly created here.

**Comment by niccokunzmann** [comment]
@Ploypiti Would you like to add a `;` to the vermaTation.js file in line 19? This would make the build green ad we know that new pull requests should also be green and not introduce errors.

File: tiles/rooms/floor/staircase.svg [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	<defs>
+		<path id="path3908" inkscape:connector-curvature="0" d="M420.733,202.206L215.272,79.612L9.812,202.206l205.461,122.595
+			L420.733,202.206z"/>
+	</defs>
+	<clipPath id="path3908_1_">
+		<use xlink:href="#path3908"  overflow="visible"/>
+	</clipPath>
+	<g transform="matrix(1 0 0 1 9.536743e-007 -2.384186e-007)" clip-path="url(#path3908_1_)">
+		
+			<image overflow="visible" width="576" height="576" xlink:href="../../../../Desktop/staircase.png"  transform="matrix(0.7352 0.0147 -0.0147 0.7352 11.2407 -2.3457)">
```
Comment (niccokunzmann) [inline tiles/rooms/floor/staircase.svg]: just saw this: ../../../../Desktop/staircase.png will not work here. Could you put the staircase.png next to this file and change this to ./staircase.png?

Comment (niccokunzmann) [reply tiles/rooms/floor/staircase.svg]: I think it is a good idea not to embed since it decreases the size.

Comment (Ploypiti) [reply tiles/rooms/floor/staircase.svg]: > Could you put the staircase.png next to this file and change this to ./staircase.png?

Seems like it didn't work, how do I keep IMAGE as svg without embedding?

**Comment by Ploypiti** [comment]
@niccokunzmann Fixed, can you merge please?

**Comment by niccokunzmann** [comment]
@Ploypiti Thanks for your pull request!

**Comment by niccokunzmann** [comment]
@meets2tarun I do not require squashing since
- this repository is also for beginners who just use the web page
- we can squash as maintainers
- to revert small changes, it might be better to have small commits, i.e. if someone wanted to revert adding the picture it should not mean breaking the build, too.

**Pull request merged**